ProjectLighthouse.Servers.GameServer/Controllers/NewsController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using LBPUnion.ProjectLighthouse.Serialization;
+using static LBPUnion.ProjectLighthouse.Types.News.News;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LBPUnion.ProjectLighthouse.Controllers;
+
+[ApiController]
+[Route("LITTLEBIGPLANETPS3_XML/news")]
+[Produces("text/xml")]
+public class NewsController : ControllerBase
```

Comment:
Not a fan of news being re-implemented here in a scuffed state; I would rather this be added when complete.

Reply:
oh that was for testing, I'll remove it